scripts/gitlab/lib.sh

```diff
@@ -96,3 +96,79 @@ curl -XPOST -d "$1" "https://matrix.parity.io/_matrix/client/r0/rooms/$2/send/m.
 # Pretty-printing functions
 boldprint () { printf "|\n| \033[1m%s\033[0m\n|\n" "${@}"; }
 boldcat () { printf "|\n"; while read -r l; do printf "| \033[1m%s\033[0m\n" "${l}"; done; printf "|\n" ; }
+
+prepare_git() {
+  # Set the user name and email to make merging work
+  git config --global user.name 'CI system'
+  git config --global user.email '<>'
+}
+
+prepare_substrate() {
+  pr_companion=$1
+  boldprint "companion pr specified/detected: #${pr_companion}"
+
+  # Clone the current Substrate master branch into ./substrate.
+  # NOTE: we need to pull enough commits to be able to find a common
+  # ancestor for successfully performing merges below.
+  git clone --depth 20 https://github.com/paritytech/substrate.git
+  previous_path=$(pwd)
+  cd substrate
+  SUBSTRATE_PATH=$(pwd)
+
+  git fetch origin refs/pull/${pr_companion}/head:pr/${pr_companion}
+  git checkout pr/${pr_companion}
+  git merge origin/master
+
+  cd "$previous_path"
```

Comment:
```suggestion
  cd substrate || exit 1
  SUBSTRATE_PATH=$(pwd)

  git fetch origin "refs/pull/${pr_companion}/head:pr/${pr_companion}"
  git checkout "pr/${pr_companion}"
  git merge origin/master

  cd "$previous_path" || exit 1
```
Just fixes a couple of shellcheck nags:
- [SC2164](https://github.com/koalaman/shellcheck/wiki/SC2164)
- [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)